core/shared/src/main/scala/mcts/Game.scala

```diff
@@ -0,0 +1,39 @@
+package mcts
+
+/**
+  * A game specification is characterized by three types.
+  *
+  * Note that by abstracting over these types, we guarantee
+  *  that the solver won't introspect them.
+  *
+  * @tparam State  Current state of game
```

Comment:
Blir det lettere for java-folket å forstå hvis du bruker A istedet for Action?
Action kan lett misforstås som en konkret type

Reply:
Jeg har lurt veldig på dette selv. Jeg begynte selvsagt med S, A, P for min egen del, men vet av erfaring at folk sliter litt med type-parametere så jeg skrev de helt ut. Men det er jo ikke helt idiomatisk det heller.... så der står jeg :)